comictaggerlib/taggerwindow.py

```diff
@@ -211,18 +211,20 @@ def __init__(
 
         self.setWindowIcon(QtGui.QIcon(str(graphics_path / "app.png")))
 
-        if config[0].Runtime_Options__type and isinstance(config[0].Runtime_Options__type[0], str):
-            # respect the command line option tag type
-            config[0].internal__save_data_style = config[0].Runtime_Options__type
-            config[0].internal__load_data_style = config[0].Runtime_Options__type[0]
+        # respect the command line option tag type
+        if config[0].Runtime_Options__type_modify:
+            config[0].internal__save_data_style = config[0].Runtime_Options__type_modify
+        if config[0].Runtime_Options__type_read:
+            config[0].internal__load_data_style = config[0].Runtime_Options__type_read
 
         for style in config[0].internal__save_data_style:
             if style not in metadata_styles:
                 config[0].internal__save_data_style.remove(style)
-        if config[0].internal__load_data_style not in metadata_styles:
-            config[0].internal__load_data_style = list(metadata_styles.keys())[0]
+        for style in config[0].internal__load_data_style:
+            if style not in metadata_styles:
+                config[0].internal__load_data_style.remove(style)
         self.save_data_styles: list[str] = config[0].internal__save_data_style
-        self.load_data_style: str = config[0].internal__load_data_style
+        self.load_data_styles: list[str] = list(reversed(config[0].internal__load_data_style))
```

Comment:
This shouldn't be reversed here. Why is it being saved in the wrong order

Reply:
Reverted.

I thought it made sense to save in the user facing order but it doesn't really.